ipld/amt/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
 name = "ipld_amt"
 description = "Sharded IPLD Array implementation."
-# TODO bump the major version when releasing v2 actors
-version = "0.2.0"
+# TODO release
+version = "1.0.0"
```

Comment:
Confused about the versioning. V2 AMT was 0.2, but V3 AMT will be 1.0?

Reply:
v0 and v2 actors' amt will use `0.2`, v3 actors will use `1.0`, does that work? No need to have multiple major versions for the same functionality

Reply:
I'm ok with it 👍